shimmer/learn/shimmer-questions.json

```
@@ -0,0 +1,83 @@
+[
+    {
+      "questionText": "The staging and innovation network of IOTA is called ...?",
+      "answerOptions": [
+        { "answerText": "Assembly", "isCorrect": false },
+        { "answerText": "Soon Network", "isCorrect": false },
+        { "answerText": "IOTA 2.0", "isCorrect": false },
+        { "answerText": "Shimmer", "isCorrect": true }
+      ]
+    },
+    {
+      "questionText": "What's the beginning of a Shimmer Testnet address?",
+      "answerOptions": [
+        { "answerText": "smrtest", "isCorrect": false },
+        { "answerText": "rms", "isCorrect": true },
+        { "answerText": "shmtest", "isCorrect": false },
+        { "answerText": "mhs", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "How many bytes can be used for the immutable data output, which is used for example for NFT Metadata?",
+      "answerOptions": [
+        { "answerText": "1337", "isCorrect": false },
+        { "answerText": "4096", "isCorrect": false },
+        { "answerText": "8192", "isCorrect": true },
+        { "answerText": "16384", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "Which TIP describes the NFT metadata standard? ",
+      "answerOptions": [
+        { "answerText": "TIP-23", "isCorrect": false },
+        { "answerText": "TIP-27", "isCorrect": true },
+        { "answerText": "TIP-33", "isCorrect": false },
+        { "answerText": "TIP-37", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "Where can you get free Testnet tokens?",
+      "answerOptions": [
+        { "answerText": "Bank", "isCorrect": false },
+        { "answerText": "Faucet", "isCorrect": true },
+        { "answerText": "Teapot", "isCorrect": false },
+        { "answerText": "Exchange", "isCorrect": false }
+      ]
+    },
+    {
+      "questionText": "What is a DEX?",
+      "answerOptions": [
+        { "answerText": "Dungeon Entry 10", "isCorrect": false },
+        { "answerText": "Disturbed Experience", "isCorrect": false },
+        { "answerText": "Decentralised Exchange", "isCorrect": true },
```

```suggestion
        { "answerText": "Decentralized Exchange", "isCorrect": true },
```

We converged to American spelling.